pytorch_lightning/metrics/classification/helpers.py

```diff
@@ -16,7 +16,16 @@
 import torch
 
 from pytorch_lightning.metrics.utils import to_onehot, select_topk
+from pytorch_lightning.utilities import LightningEnum
 
+class DataType(LightningEnum):
+    """
+    Enum to represent data type
+    """
+    BINARY = 'binary'
+    MULTILABEL = 'multilabel'
+    MULTICLASS = 'multiclass'
+    MDMC = 'mdmc' # multi-dim multi-class
```

Comment:
can we maybe write this out? especially if we use an enum, it should be descriptive.

Reply:
Perhaps
```
class DataType(Enum):
    BINARY = 'binary'
    MULTILABEL = 'multi-label'
    MULTICLASS = 'multi-class'
    MDMC = 'multi-dim multi-class'
```
(not from LightningEnum)
better, not allowing string comparison? Otherwise, `DataType.MULTILABEL == 'multi-label'` doesn't seem natural.

Reply:
It specifically goes for the last part of MDMC, i'm thinking if we can write this out. Shouldn't be to hard. I would stick to LightningEnum if possible

Reply:
ah, you meant something like
```
class DataType(LightningEnum):
...
MULTIDIM_MULTICLASS = "multi-dim multi-class"
```

Reply:
yes